src/routers/alpha-router/alpha-router.ts

```diff
@@ -1125,7 +1132,9 @@ export class AlphaRouter
       return swapRouteWithSimulation;
     }
 
-    return swapRoute;
+    // anytime we don't have a simulated gasLimit estimate
+    // we set simulationError to true
+    return { ...swapRoute, simulationError: true };
```

Comment:
I don't think this is right, I think we should just leave it undefined if we didn't actually run a simulation

Reply:
sure